core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -123,6 +123,10 @@ abstract class GoogleServicesDAO(groupsPrefix: String) extends ErrorReportable {
     */
   def listBillingAccountsUsingServiceCredential(implicit executionContext: ExecutionContext): Future[Seq[RawlsBillingAccount]]
 
+  def getProjectBillingAccount(projectName: RawlsBillingProjectName, userInfo: UserInfo)(implicit executionContext: ExecutionContext): Future[Option[String]]
+
+  def getRootBillingAccount(billingAccountId: String, userInfo: UserInfo)(implicit executionContext: ExecutionContext): Future[Option[String]]
```

Comment:
Could we comment on this explaining/linking to https://cloud.google.com/billing/docs/concepts#subaccounts .  It's a thing I hadn't really realized existed until recently and certainly it's relevant here.  

Furthermore, I don't think we need the parent/root account.  My understanding from talking with Google and one of the 3rd party vendors managing Billing Accounts for the NIH is that when the BiqQuery spend data export is set up, it happens at the subaccount level.  I do not know if it needs to happen at the subaccount or if it can happen for both.   At the very least, what I can tell you is that for a subset of Billing Accounts, the data exporting is going to be at the subaccount level.  I wouldn't be surprised if the Broad's Billing Account is set up to export at the root level, but I don't know.

Reply:
Once again, I would rename the method to be specific about whether it is returning the Root Billing Account's name or ID.  It is not clear from reading the implementation.

Reply:
You're right that the root account isn't actually needed. Per documentation [here](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#data-schema)